beacon-chain/blockchain/process_attestation_test.go

```diff
@@ -491,6 +491,38 @@ func TestVerifyLMDFFGConsistent_OK(t *testing.T) {
 	assert.NoError(t, err, "Could not verify LMD and FFG votes to be consistent")
 }
 
+func TestVerifyFinalizedConsistency_OK(t *testing.T) {
```

Comment:
Please also test the unhappy path.

Reply:
Unhappy path was tested in this func but I'll move it to a new test for clarity

See line 518